### PR TITLE
Grid layout

### PR DESCRIPTION
**Pull request by briangerez**
###Summary
 Grid layout
###Screenshot
image

###Trello card

https://trello.com/c/iFoMjczM/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -11,31 +7,33 @@
...
-.tech {
-  @extend %tech-border-radius;
+%tech {
   @extend %size-tech;
   align-items: center;
-  background-color: $earls-green;
   display: flex;
   flex-direction: column;
-  left: 45px;
   margin: 15px 15px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: use shorthand for this margin

File: scss/tech.scss [hunk 2]
```diff
@@ -11,31 +7,33 @@
...
   position: relative;
   padding: 0 25px 0 5px;
+}
+
+.tech {
+  @extend %tech;
```
Comment (FrankIglesias) [inline scss/tech.scss]: why do you extend tech from %tech?... you can do this in html like `first-tech tech`

File: scss/tech.scss [hunk 3]
```diff
@@ -11,31 +7,33 @@
...
-  &::before {
-    @extend %tech-border-radius;
-    @extend %size-tech;
+.first-tech {
+  @extend %tech;
+  align-self: center;
+  grid-column: 1 / 2;
+  grid-row: 1 / 3;
+  justify-self: end;
+  &:after {
```
Comment (FrankIglesias) [inline scss/tech.scss]: add empty line here

File: scss/tech.scss [hunk 4]
```diff
@@ -11,31 +7,33 @@
...
     content: "";
-    border: 2px solid $earls-green;
-    left: -10px;
+    border: 1px solid $earls-green;
+    height: 670px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: height 100%?

File: scss/tech.scss [hunk 5]
```diff
@@ -11,31 +7,33 @@
...
     position: absolute;
-    top: -10px;
-    transition: all 0.5s ease-in-out;
-  }
-
-  &:hover::before {
-    transform: translateX(20px) translateY(20px);
+    right: 0;
+    top: -180px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: why this?

File: scss/tech.scss [hunk 6]
```diff
@@ -67,7 +65,6 @@
 }
 
 .tech-container {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: [first] 50% [line2] 25% [line3] 25%  ;
```
Comment (FrankIglesias) [inline scss/tech.scss]: you can use also fr notation but its ok

File: scss/tech.scss (hunk 4 above)
Comment (briangerez) [reply scss/tech.scss]: Use this for build the green line.

File: scss/tech.scss (hunk 5 above)
Comment (briangerez) [reply scss/tech.scss]: previus idem.

**Pull request merged**